docs/umlDiagrams/DeleteFlashcard.puml

```diff
@@ -0,0 +1,57 @@
+@startuml
+'https://plantuml.com/class-diagram
+
+!define LOGIC_COLOR #3333C4
+!define LOGIC_COLOR_T1 #7777DB
+!define LOGIC_COLOR_T2 #5252CE
+!define LOGIC_COLOR_T3 #1616B0
+!define LOGIC_COLOR_T4 #101086
+
+participant "<<Class>> :Clanki" as Main
+participant ":Ui" as Ui
+participant ":Parser" as Parser
+participant ":Delete Command" as DeleteCommand
```

Comment:
Is "Delete Command" a class? maybe it is called DeleteCommand? I think classes should have exact name on sequence diagram